resources/compass/values.yaml

```diff
@@ -1,4 +1,7 @@
 global:
+  defaultTenant : 3e64ebae-38b5-46a0-b1ed-9ccee153a0ae
+  tenants: foo bar
```

Comment:
What does the `foo bar` mean? Tenants should be in UUID format

Reply:
Those are sample tenants for demonstrating multi-tenancy model. Those will be converted to UUID on the flight.